lib/ts/signals.cc

```diff
@@ -46,7 +46,7 @@ signal_check_handler(int signal, signal_handler_t handler)
     sigact = (void *)oact.sa_sigaction;
   }
 
-  if (sigact != handler) {
+  if (sigact != (void *)handler) {
```

Comment:
I did look / try not using void *, but it turns out the sigaction is not the same type as the handler. So some casting would be needed.

And yeh, I'll change to amc style cast, only because you are the best, and currently sitting in the #1 spot on the ranking.